File: Back/Api/Domain/Lot.cs


namespace Domain
{
    public class Lot
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public int EventId { get; set; }
        public DateTime? DateInicial { get; set; }
        public DateTime? DateFinal { get; set; }
        public DateTime CreatedAt { get; set; }

        public Event Event { get; set; }
    }
}

File: Front/Events-APP/src/app/models/Lot.ts
import { Event } from "./Event";

export interface Lot {
    id: number;
    name: string; 
    price: number; 
    quantity: number;
    eventId: number;
    dateInicial?: Date; 
    dateFinal?: Date; 
    createdAt: Date; 
    event: Event;
}
